ngraph/core/src/op/maximum.cpp

```diff
@@ -58,7 +58,7 @@ namespace maximumop
 
 // ------------------------------------ v1 -------------------------------------
 
-constexpr NodeTypeInfo op::v1::Maximum::type_info;
+NGRAPH_RTTI_DEFINITION(op::v1::Maximum, "Maximum", 1);
```

Comment:
```suggestion
NGRAPH_RTTI_DEFINITION(op::v1::Maximum, "Maximum", 1, BinaryElementwiseArithmetic);
```
And for other operations too

Reply:
Done